docs/en/cloud/manage/openapi.md

```diff
@@ -0,0 +1,113 @@
+
+
+| Method | Endpoint                                                                    | Description                                         |
+| ------ | --------------------------------------------------------------------------- | --------------------------------------------------- |
+| GET    | /v1/organizations/`organizationId`/keys                                     | Get list of API keys                                | 
+| POST   | /v1/organizations/`organizationId`/keys                                     | Create a new key                                    |  
+| GET    | /v1/organizations/`organizationId`/keys/`keyId`                             | Get API key details                                 |
+| PATCH  | /v1/organizations/`organizationId`/keys/`keyId`                             | Update API key details                              |
+| DELETE | /v1/organizations/`organizationId`/keys/`keyId`                             | Delete an API key                                   |
+
+## Members and Invitations
+
+Members are individuals with access to your ClickHouse Cloud account. Invitiations reflect individuals that were added by an admin, but have yet to sign into ClickHouse Cloud to accept the invitation and become a member.
+
+
+| Method | Endpoint                                                                    | Description                                         |
+| ------ | --------------------------------------------------------------------------- | --------------------------------------------------- |
+| GET    | /v1/organizations/`organizationId`/members                                  | List organization members                           | 
+| GET    | /v1/organizations/`organizationId`/members/`userId`                         | Get an organization member's  details               |
+| PATCH  | /v1/organizations/`organizationId`/members/`userId`                         | Update an organization member's details             |
+| DELETE | /v1/organizations/`organizationId`/members/`userId`                         | Remove an organization member                       |
+| GET    | /v1/organizations/`organizationId`/invitations                              | List invitations                                    | 
+| POST   | /v1/organizations/`organizationId`/invitations                              | Create invitations                                  |
+| GET    | /v1/organizations/`organizationId`/invitations/`invitationId`               | Get invitation details                              | 
+| DELETE | /v1/organizations/`organizationId`/invitations/`invitationId`               | Delete invitation                                   | 
+
+## Activities
+
+Activities refer to the [audit log](/docs/en/cloud/security/organization-activity) within your ClickHouse Cloud account. This does not include database activities.
```

Comment:
What file is this supposed to be?   It has to be the name of the markdown file, not the slug.

Reply:
TY! I'll update it. Still need to QA with the internal team before merging and will ping when ready to go live.